src/service/leetcode.rs

```diff
@@ -301,11 +301,71 @@ impl<'a> ServiceProvider<'a> for Leetcode<'a> {
         Ok(problems_res)
     }
 
-    fn list_problems(&mut self, list: List) -> Result<()> {
+    fn list_problems_with_tag(&mut self, list: List) -> Result<()> {
+        let query = r#"
+            query getTopicTag($slug: String!) {
+                 topicTag(slug: $slug) {
+                   name
+                   translatedName
+                   slug
+                   questions {
+                     questionId
```

Comment:
Probably we want frontendQuestionId?

Reply:
Hmm, when I try frontendQuestionId I get a 400 error. Any idea if there is another name for this node?

Reply:
Try `questionFrontendId`

Reply:
Worked! Thanks